src/main/java/com/epam/lab/paymentsystem/service/impl/AccountServiceImpl.java

```diff
@@ -48,6 +48,9 @@ public Account createAccount(AccountDto accountDto) throws UnsupportedOperationE
 
     String login = userService.getCurrentUserLogin();
     User user = userService.getUserByLogin(login);
+    if (!user.getRole().getRoleStatus().toString().equals("USER")) {
```

Comment:
Please, move check in separate method. So it will be like `if (!userRoleStatusEquals("USER"))`